src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,64 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
+    @Column(name = "password", length = 102500)

```

Comment:
The `@Column` annotation for `password` specifies a length of 102500, which seems excessively large for a password field. Consider reducing this length to a more reasonable size, such as 255, unless there is a specific requirement for such a large length.

Reply:
but, the column for password, needs to be larger than 255,